### PR TITLE
update README using github markdown format

### PR DESCRIPTION
**Pull request by za**
I've updated the README with:
- environment setup
- installation
- run
- contribute

Just in case someone need it and might want to join hacking this app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,240 @@
 # django-dequorum
-A simple web forum for Django
+A simple web forum Django application
+
+# Environment setup
+
+1. Setup [virtualenv](https://pypi.python.org/pypi/virtualenv) with python3 ([virtualenvwrapper](https://pypi.python.org/pypi/virtualenvwrapper/) installed)
+
+    ``` 
+    $ mkvirtualenv --python=/usr/bin/python3 django-python3
```
Comment (funkybob) [inline README.md]: Usually "-p python3" is enough... or "-p `which python3`" to be sure.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,240 @@
 # django-dequorum
-A simple web forum for Django
+A simple web forum Django application
+
+# Environment setup
+
+1. Setup [virtualenv](https://pypi.python.org/pypi/virtualenv) with python3 ([virtualenvwrapper](https://pypi.python.org/pypi/virtualenvwrapper/) installed)
```
Comment (funkybob) [inline README.md]: There's no particular reason to use virtualenvwrapper in this case.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,240 @@
...
+    ```
+
+   To start working on virtualenv
+
+    ```
+    $ workon django-python3
+    (django-python3) ~/
+    ```
+
+    If you are using [zsh](http://www.zsh.org/) there will be a `(django-python3)` at your terminal indicating you're on `django-python3` virtualenv.
```
Comment (funkybob) [inline README.md]: The prompt change is not specific to zsh.

**Comment by za** [comment]
Updated based on comments.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,275 @@
...
+
+    ```
+    (django-python3) ~$ deactivate
+    ```
+
+Notes: 
+
+* `(django-python3)` prompt change at your terminal indicating you're on `django-python3` virtualenv.
 
 # Installation
```
Comment (funkybob) [inline README.md]: Perhaps rename this to "Quickstart", since this isn't the general install procedure.

**Pull request merged**